### ISSUE TITLE
media queue filling

### ISSUE THREAD

**Issue by Marcioghiraldelli** [AzuraCast/AzuraCast#5645]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

xxx

### What happened?

I noticed yesterday that my azuracast, in the debugger part, doesn't stop increasing the media queue for verification.
Yesterday I left and got to be more than 70 thousand files in a queue.
So I cleared the queue, restarted the server and even so, the queue goes up again, I didn't send files to the server and even then it keeps going up this number. I did a test now sending 20 files by SFTP and it's been more than an hour with status processing file. I believe it's something connected to this number that doesn't stop growing.
Is there any command I can clear the queue and prevent this?

### Relevant log output

```shell
https://prnt.sc/_-2kUJayr1Xx
```

**Comment by xewl** [comment]
xxx is not a version.

**Comment by Marcioghiraldelli** [comment]
version  v0.17.3 Stable • Docker • PHP 8.1

**Comment by Marcioghiraldelli** [comment]
img1: https://prnt.sc/vgyJ5oS7Cz6q
img2: https://prnt.sc/g3y4WLRmZraN

I'm using a volume configuration elsewhere in the docker-compose.override.yml file.
Everything was ok and from a few days ago it started to look like this where the new files only have a status of processing file and the queue in the system debugger only increases.
Never ends, I checked the accounts and none are uploading files. I looked at the azuracast logs and didn't see anything about this issue.
